### PR TITLE
[cisco_aironet]: handle CLIENT_ORCH_LOG messages

### PR DESCRIPTION
**Pull request by pkoutsovasilis**
## Proposed commit message

This PR fixes parsing of CLIENT_ORCH_LOG in cisco aironet integration

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] ~~I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).~~

## Author's Checklist

N/A

## Related issues

- Closes https://github.com/elastic/integrations/issues/8828

## Screenshots

N/A

### CODE REVIEW

File: packages/cisco_aironet/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.13.1"
+  changes:
+    - description: Fix CLIENT_ORCH_LOG messages.
+      type: bugfix
+      link: https://github.com/elastic/integrations/issues/8828
```
Comment (mjwolf) [inline packages/cisco_aironet/changelog.yml:6]: This should be the PR url

File: packages/cisco_aironet/data_stream/log/_dev/test/pipeline/test-aironet-messages.log-expected.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
     "expected": [
         {
-            "@timestamp": "2023-09-06T23:08:09.371Z",
+            "@timestamp": "2024-09-06T23:08:09.371Z",
```
Comment (mjwolf) [inline packages/cisco_aironet/data_stream/log/_dev/test/pipeline/test-aironet-messages.log-expected.json:4]: Why did you change the year? The tests should have continued to pass with the dynamic field. It doesn't really matter much though

**Review by mjwolf — COMMENTED** [review]
LGTM, other than the changelog link

File: packages/cisco_aironet/changelog.yml (hunk 1 above)
Comment (pkoutsovasilis) [reply packages/cisco_aironet/changelog.yml:6]: hmmm I thought it was equivalent but sure I can change it

File: packages/cisco_aironet/data_stream/log/_dev/test/pipeline/test-aironet-messages.log-expected.json (hunk 2 above)
Comment (mjwolf) [reply packages/cisco_aironet/data_stream/log/_dev/test/pipeline/test-aironet-messages.log-expected.json:4]: Ah, ok, I didn't know that was autogenerated

**Review by mjwolf — APPROVED** [review]
